monai/apps/pathology/transforms/spatial/dictionary.py

```diff
@@ -112,7 +110,9 @@ def __init__(
     def randomize(self, data: Any = None) -> None:
         self.seed = self.R.randint(10000)  # type: ignore
 
-    def __call__(self, data: Mapping[Hashable, np.ndarray]) -> Union[Dict[Hashable, np.ndarray], List[Dict]]:
+    def __call__(
```

Comment:
Please add the backend for dict transform also.

Thanks.

Reply:
Thanks! Done!